src/components/CommandItem/__tests__/CommandItem.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { cleanup, render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import CommandItem from '../CommandItem';
+
+afterEach(cleanup); // eslint-disable-line
+
+describe('commandItem', () => {
+  it('should render component with empty entity', () => {
+    const tree = renderer.create(<CommandItem entity={{}} />).toJSON();
+    expect(tree).toMatchInlineSnapshot(`
+      <div
+        className="str-chat__slash-command"
+      >
+        <span
+          className="str-chat__slash-command-header"
+        >
+          <strong />
+           
+        </span>
+        <br />
+        <span
+          className="str-chat__slash-command-description"
+        />
+      </div>
+    `);
+  });
+
+  it('should render component with custom entity prop', async () => {
+    const entity = { name: 'name', args: 'args', description: 'description' };
+    const Component = <CommandItem entity={entity} />;
+
+    const { getByText } = render(Component);
+    await waitFor(() => {
```

Comment:
Do you need the waitFor here? I was under the impression that `render` is synchronous

Reply:
you're right it's not needed